datafusion/physical-expr/src/string_expressions.rs

```diff
@@ -643,6 +644,31 @@ pub fn overlay<T: OffsetSizeTrait>(args: &[ArrayRef]) -> Result<ArrayRef> {
     }
 }
 
+///Returns the Levenshtein distance between the two given strings
+/// LEVENSHTEIN('kitten', 'sitting') = 3
+pub fn levenshtein<T: OffsetSizeTrait>(args: &[ArrayRef]) -> Result<ArrayRef> {
```

Comment:
Would it be possible to use the existing implementation in : https://github.com/apache/arrow-datafusion/blob/aaaf698b5e46dd1c4d29bce69dc539a667d2dda6/datafusion/common/src/utils.rs#L484